website/thabloid/apps.py

```diff
@@ -21,3 +22,20 @@ def menu_items(self):
                 }
             ],
         }
+
+    def user_profile_form_fields(self, instance=None):
+        from thabloid.models import ThabloidUser
```

Comment:
Do these really need to be deferred imports? I think it shouldn't be required for apps to be initialized jsut to import models